commands/databasescommands/backupcommands/restore.go

```diff
@@ -0,0 +1,135 @@
+package backupcommands
+
+import (
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/db/v1/instances"
+	"github.com/rackspace/rack/util"
+)
+
+var restore = cli.Command{
+	Name:        "restore",
+	Usage:       util.Usage(commandPrefix, "create", "[--id <id> | --stdin id] --name <instanceName> --flavor <flavorId> --size <instanceSize> --wait-for-completion"),
```

Comment:
s/create/restore
s/flavor/flavor-id (and maybe add a flavor-name parameter?)

By convention, we only include (potentially) required parameters in the `Usage` string, so we can drop `--wait-for-completion` here.